### PR TITLE
homework folder and html forms

### PR DESCRIPTION
**Pull request by NFartenadze**

### CODE REVIEW

**Comment by NFartenadze** [comment]
I didn't see task file somehow :') I will delete this and do the task according to requirements

File: homeworks/nika-fartenadze/01-html/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      <section>
+        <h1>Hi, I am Nika Fartenadze,<br />Front-end Developer</h1>
+        <div>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Iusto,
+            quia.
+          </p>
+        </div>
+      </section>
+      <!-- i am not sure if setting width/height attributes is considered css :(, it didn't take entire page without them )-->
```
Comment (JulianOviedo) [inline homeworks/nika-fartenadze/01-html/index.html:26]: its ok ! dont worry

File: homeworks/nika-fartenadze/01-html/contact.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (JulianOviedo) [inline homeworks/nika-fartenadze/01-html/contact.html:6]: add title

**Review by JulianOviedo — APPROVED** [review]
why ds_store files ? delete them please

**Comment by JulianOviedo** [comment]
> I didn't see task file somehow :') I will delete this and do the task according to requirements

image

**Review by JulianOviedo — CHANGES_REQUESTED** [review]
delete ds_store unnecesary files

**Comment by NFartenadze** [comment]
I also removed ds_store files, i just forgot to commit them separately

**Review by JulianOviedo — APPROVED** [review]

**Pull request merged**